client/lib/google-apps/index.js

```diff
@@ -1,26 +1,19 @@
 /**
- * External dependencies
+ * Internal dependencies
  */
-import { numberFormat } from 'i18n-calypso';
+import formatCurrency from 'lib/format-currency';
 
 /**
  * Constants
  */
 const GOOGLE_APPS_LINK_PREFIX = 'https://mail.google.com/a/';
-const PRICE_FORMAT = /(\d+[.,]?\d+)/;
 
-function getAnnualPrice( price ) {
-	return price && price.replace( PRICE_FORMAT, ( value ) => {
-		const number = parseFloat( value );
-		return number % 1 === 0 ? number : numberFormat( number, 2 );
-	} );
+function getAnnualPrice( cost, currencyCode ) {
+	return formatCurrency( cost, currencyCode, { precision: 0 } );
```

Comment:
I am not sure we should use precision 0 here. The annual price in NZD is NZ$72.5, but this will show up as NZ$73 in the details section of the component, but as NZ$72.5 in the shopping cart.